views/App.jsx

```diff
@@ -35,35 +37,29 @@ const App = (props) => {
   }, [])
 
   return (
-    <MuiThemeProvider>
-      <CssBaseline />
-      <ThemeContext.Provider
-        value={{ classes: props.classes, theme: props.theme }}
-      >
-        <DimensionsContext.Provider value={[width, setWidth]}>
-          <SidebarContext.Provider value={[openSidebar, setOpenSidebar]}>
-            <NotificationContext.Provider
-              value={[notification, setNotification]}
-            >
-              <ConfigurationsContext.Provider
-                value={[configurations, setConfigurations]}
-              >
-                <AuthContext.Provider value={[user, setUser]}>
-                  <Router {...props} user={user} setUser={setUser} />
-                  <Snackbar
-                    open={notification.open}
-                    message={notification.message}
-                    autoHideDuration={2000}
-                    onClose={handleNotificationClose}
-                  />
-                </AuthContext.Provider>
-              </ConfigurationsContext.Provider>
-            </NotificationContext.Provider>
-          </SidebarContext.Provider>
-        </DimensionsContext.Provider>
-      </ThemeContext.Provider>
-    </MuiThemeProvider>
+    <DimensionsContext.Provider value={[width, setWidth]}>
```

Comment:
I wonder if we can remove this context altogether and rely on media queries

Reply:
[6b310d1](https://github.com/AMP-SCZ/dpdash/pull/458/commits/6b310d18b25129974d8ce470610170164ed5d426)